myuw_vue/components/home/summaries.vue

```diff
@@ -177,10 +182,15 @@ export default {
       isLibraryReady: 'isReady',
       isLibraryErrored: 'isErrored',
     }),
+    loadData() {
+      return (this.alum || this.student || this.past_stud || this.employee || this.past_emp);
```

Comment:
Husky card only shows for student, past_stud, employee, or past_employee. We should match that on home page. This reduces unnecessary requests to HFS and Lib account apis.